app/views/cases/ico/_new_form.html.slim

```diff
@@ -1,3 +1,5 @@
+= f.text_field :ico_officer_name
```

Comment:
I thought you were going to play back this field on the case details page as part of this PR.

Reply:
Ya, but I guess the case details page wasn't ready yet, not fussed either way.

Reply:
I've added this now.